uyuni-docs-helper

```diff
@@ -141,6 +141,14 @@ if [ -z ${PRODUCT} ]; then
     print_incorrect_syntax "Product (-p) is mandatory"
 fi
 
+# Check if SELinux is present and Enforcing
+if [ -f /etc/selinux/config ]; then
```

Comment:
Does having this config file mean SELinux is in enforcing mode? I'd expect it exists even in permissive mode.

Reply:
The line 145 checks if SELinux is enabled at all 🙂 
The line 146 checks if SELinux is in Enforcing mode

Reply:
Ouch, of course, that's calling a command.

Reply:
Yeah also was a bit confused in the first place how to best programmatically check for SELinux being enabled, but it works pretty well that way 🙂